tools/mo/openvino/tools/mo/middle/passes/conv.py

```diff
@@ -16,8 +16,11 @@
 
 def pad_op_transform(graph: Graph, match: dict):
     op = match['op']
-    pad_op = match['pad_op']
-    input_data = pad_op.in_node(0)
+    pad_op: Node = match['pad_op']
+
+    # to keep reshape-ability if Pad receives pads_begin/pads_end from shape subgraph
+    if pad_op.in_port(1).get_source().node.soft_get('can_be_fused') is False:
```

Comment:
use method has_and_set('can_be_fused')

Reply:
in most of the cases `soft_get` is used
![image](https://user-images.githubusercontent.com/5703039/159681291-4b6e2ff2-d785-4fa0-a64a-14ef02b3f33f.png)

If we use `has_and_set` as below
```
if not pad_op.in_port(1).get_source().node.has_and_set('can_be_fused'):
   return
```
this will be incorrect: it will return even in cases there Pad should be fused. 'can_be_fused` is not always set, and if it's not set explicitly to `False` then it means that it can be fused (default value is True). Therefore I left `soft_get`